### PR TITLE
add code formatting to commands in expose-external-ip-address tutorial

### PR DESCRIPTION
**Pull request by RA489**
add code formatting to commands in expose-external-ip-address tutorial
fixes https://github.com/kubernetes/website/issues/24873

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25080--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/expose-external-ip-address/

File: content/en/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 1]
```diff
@@ -61,26 +49,36 @@ The preceding command creates a
 
 1. Display information about the Deployment:
 
+        ```shell
```
Comment (kbhawkey) [inline content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:52]: @RA489 , Would you remove the indentation of this code block and check the other code blocks on the page?
Thanks!

Comment (tengqm) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:52]: Not sure if we want to make this part of the content style guideline. Marking code blocks using indentation is fragile. Maybe we should promote explicit code block marking?

Comment (RA489) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:52]: @kbhawkey sure i will check and remove.

Comment (RA489) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md]: @kbhawkey removed the identation of the code.

Comment (kbhawkey) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md]: 👀

File: content/en/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 2]
```diff
@@ -148,29 +156,27 @@ The preceding command creates a
...
+    ```shell
+    Hello Kubernetes!
+    ```
 
 ## {{% heading "cleanup" %}}
 
-
 To delete the Service, enter this command:
 
+    ```shell
```
Comment (kbhawkey) [inline content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:167]: Hi @RA489 . The updates look great! Can you remove the indentation for the last two code blocks?
See, https://deploy-preview-25080--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/expose-external-ip-address/#cleaning-up

Comment (RA489) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:167]: sure

File: content/en/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 3]
```diff
@@ -61,26 +49,36 @@ The preceding command creates a
 
 1. Display information about the Deployment:
 
-        kubectl get deployments hello-world
-        kubectl describe deployments hello-world
+    ```shell
+    kubectl get deployments hello-world
+    kubectl describe deployments hello-world
+    ```
```
Comment (tengqm) [inline content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:55]: The above lines can be aligned with the first character in line 50, i.e. 'D'.

File: content/en/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 4]
```diff
@@ -61,26 +49,36 @@ The preceding command creates a
...
 
 1. Display information about your ReplicaSet objects:
 
-        kubectl get replicasets
-        kubectl describe replicasets
+    ```shell
+    kubectl get replicasets
+    kubectl describe replicasets
+    ```
```
Comment (tengqm) [inline content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:62]: Similarly, the above three lines can be indented by 3 spaces.

File: content/en/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 5]
```diff
@@ -61,26 +49,36 @@ The preceding command creates a
...
 
 1. Create a Service object that exposes the deployment:
 
-        kubectl expose deployment hello-world --type=LoadBalancer --name=my-service
+    ```shell
+    kubectl expose deployment hello-world --type=LoadBalancer --name=my-service
+    ```
```
Comment (tengqm) [inline content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:68]: here and the following.

File: content/en/docs/tutorials/stateless-application/expose-external-ip-address.md (hunk 2 above)
Comment (tengqm) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:167]: No indentation for  the above code block is needed.
Indentation by 4 spaces is an old practice for code hilighting.

Comment (RA489) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md]: @kbhawkey removed the indentation for the last two code blocks.

**Comment by kbhawkey** [comment]
Thanks @RA489 for working on this!
/lgtm

File: content/en/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 6]
```diff
@@ -50,7 +39,6 @@ external IP address.
 kubectl apply -f https://k8s.io/examples/service/load-balancer-example.yaml
 ```
```
Comment (kbhawkey) [inline content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:40]: 👋 @RA489 . I just looked at the preview and noticed that the first item in the list does not indent the "codenew" shortcode under the item starting with "Run a Hello World ...". The code block following this shortcode should also be indented under "Run a Hello World ...". It appears that the numbered list is not rendering completely.
I suggest to check the indentation of line 42 which starts with, "The preceding command creates ...".

Comment (RA489) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:40]: sure

Comment (RA489) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:40]: I will update accordingly.

Comment (RA489) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:40]: @kbhawkey updated the indentation. PTAL.

File: content/en/docs/tutorials/stateless-application/expose-external-ip-address.md (hunk 3 above)
Comment (tengqm) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:55]: Pls indent line 52-55 by three spaces.

**Comment by RA489** [comment]
@kbhawkey updated the identation PTAL.

File: content/en/docs/tutorials/stateless-application/expose-external-ip-address.md (hunk 6 above)
Comment (tengqm) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:40]: No. Line 36 and the following lines are still not properly indented, so the result is incorrect.

https://deploy-preview-25080--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/expose-external-ip-address/

Comment (kbhawkey) [reply content/en/docs/tutorials/stateless-application/expose-external-ip-address.md:40]: @RA489 , Could you:
- indent line 36 the same way you indented line 52
- indent lines 38-40 the same way you indented line 52
These code blocks should appear under the `R` in `Run a Hello World application in your cluster:`
OK?

**Comment by kbhawkey** [comment]
Hi @RA489 .
This is very close. Let's move these changes forward.
Thanks!
/lgtm

**Pull request merged**